### PR TITLE
added weekend to availability

### PR DESCRIPTION
**Pull request by apluscs**
!
availability

### CODE REVIEW

**Review by meagancheese — CHANGES_REQUESTED** [review]
Make it Sun-Sat and add screenshot.

**Review by meagancheese — CHANGES_REQUESTED** [review]
Remove BaseDAO stuff

File: src/main/webapp/availability.html [hunk 1]
```diff
@@ -43,6 +43,77 @@ <h5 class="text-center lead">Update your availability to be an Interviewer for t
...
+          </div>
+          <div class="col">
+            <select class="form-control">
+              <option>AM</option>
+              <option selected>PM</option>
+            </select>
+          </div>
+        </div>
+      </form>
+      <br>
```
Comment (meagancheese) [inline src/main/webapp/availability.html:116]: Nit: add space after br

**Review by meagancheese — APPROVED** [review]
Add a space after the br I indicated but otherwise, LGTM

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**